src/webgpu/shader/validation/expression/call/builtin/unpack4x8snorm.spec.ts

```diff
@@ -0,0 +1,61 @@
+const kFn = 'unpack4x8snorm';
+export const description = `Validate ${kFn}`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../../../shader_validation_test.js';
+
+const kArgCases = {
+  good: '(1u)',
```

Comment:
Please add a good case for abstract-int (`1`), here and for `unpack4x8unorm`

Reply:
Done. Added abstract_int for good args.